app/config/question-bank.js

```diff
@@ -429,22 +429,65 @@ const questionBank = {
           nextUrl: 'cover',
           url: 'project-type',
           preValidationKeys: [],
+          ineligibleContent: {
+            messageContent: `
+            This grant is only for: <br> 
+            <ul class="govuk-list govuk-list--bullet">
+            <li>replacing an existing store that is no longer fit for purpose</li>
+            <li>adding a new store to increase existing capacity</li>
+            <li>expanding an existing store (for example, by adding an extra ring to a steel tank)</li>
+            </ul>
+            </br>
+            <p class="govuk-body">A store is no longer fit for purpose if it has reached the end of its design life (for example, it may be suspectable to leaks or failure).</p>
+            `,
+            messageLink: {
+              url: 'https://www.gov.uk/government/collections/rural-payments-and-grants',
+              title: 'See other grants you may be eligible for.'
+            }
+          },
           type: 'single-answer',
           minAnswerCount: 1,
+          sidebar: {
+            values: [{
+              heading: 'Eligibility',
+              content: [{
+                para:`This grant is only for:`,
+                items: [
+                        'replacing an existing store that is no longer fit for purpose',
+                        'adding a new store to increase existing capacity',
+                      'expanding an existing store (for example, by adding an extra ring to a steel tank)'
+                      ],
+                additionalPara:`A store is no longer fit for purpose if it has reached the end of its design life (for example, it may be susceptible to leaks or failure).`,
+              }]
+            }]
+          },
           validate: [
             {
               type: 'NOT_EMPTY',
-              error: 'Select when the project will have planning permission'
+              error: 'Select increase storage capacity option'
             }
           ],
           answers: [
             {
               key: 'project-type-A1',
-              value: 'Less than 6 months'
+              value: 'Replace an existing store that is no longer serviceable'
             },
             {
-              key: 'projectType'
-            }
+              key: 'project-type-A1',
+              value: 'Add a new store to increase existing capacity'
+            },
+            {
+              key: 'project-type-A1',
```

Comment:
Keys should be unique here as well

Reply:
Thanks for the review 💯 
I have just pulled the main and pushed my new changes accordingly it's ready for your review. :)